vocabularies/Org.OData.Aggregation.V1.xml

```diff
@@ -218,7 +221,14 @@
         <Parameter Name="Entity" Type="Edm.EntityType" Nullable="false" />
         <Parameter Name="Hierarchy" Type="Edm.String" Nullable="false" />
         <Parameter Name="Node" Type="Edm.PrimitiveType" Nullable="false" />
-        <Parameter Name="MaxDistance" Type="Edm.Int16" Nullable="true" />
+        <Parameter Name="MaxDistance" Type="Edm.Int16">
+          <Annotation Term="Core.OptionalParameter">
+            <Record>
+              <PropertyValue Property="DefaultValue" Int="32767"/>

```

Comment:
I hope this is sufficient for all use cases :smile:

Reply:
It's the largest Edm.Int16 there is.